cypress/commons/actions/brewery/BreweryParameters.js

```diff
@@ -115,6 +115,25 @@ function getCustomersTableCell(colName, rowIndex) {
   return TableParameters.getCell(getCustomersTable(), colName, rowIndex);
 }
 
+function getCustomersErrorsPanel() {
+  return TableParameters.getErrorPanel(getCustomersTable());
+}
+function getCustomersErrorsHeader() {
+  return TableParameters.getErrorsHeader(getCustomersTable());
+}
+function getCustomersErrorAccordions() {
+  return TableParameters.getErrorsAccordions(getCustomersTable());
+}
```

Comment:
`getCustomersErrorAccordions` should be `getCustomersErrorsAccordions` to be consistent with `getErrorsAccordions` and the other functions names